src/componenets/signupPage/index.jsx

```diff
@@ -19,13 +27,22 @@ const SignupPage = () => {
       password: password,
     };
     const result = await signUp(payload);
+    if (result.status === 200) {
+      toast.success(result.data.metadata.message);
+      setTimeout(() => {
+        navigate(path);
+      }, 2000);
```

Comment:
same as in comment in signin timeout and toast is not required in succes case directly navigate please